coordinator/src/models/errors/validation-errors.ts

```diff
@@ -84,3 +84,27 @@ export function createResourceTypeIssue(expectedResourceType: string): fhir.Oper
     diagnostics: `Incorrect FHIR resource type. Expected ${expectedResourceType}.`
   }
 }
+
+export function createTaskIncorrectValueIssue(
+  fieldName: string,
+  requiredFieldValue: string
+): fhir.OperationOutcomeIssue {
+  return {
+    severity: "error",
+    code: "value",
+    diagnostics: `Task.${fieldName} must be '${requiredFieldValue}'.`,
```

Comment:
This message won't look right when requiredFieldValue is "in-progress or rejected" because of the quotes. Could just remove the quotes, or could accept an array of acceptable field values

Reply:
I guess remove the quotes and pass quotes in to the function? `must be 'in-progress' or 'rejected'` is probably the best way to display it